run_gh.sh

```
@@ -0,0 +1,13 @@
+gh pr create --head bolt-parallelize-remote-check --title "⚡ Bolt: リモートブランチ存在確認の並列化による高速化" --body "💡 What:
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)